File: src/main/java/com/falsepattern/zigbrains/zon/parser/ZonTokenSets.java
package com.falsepattern.zigbrains.zon.parser;

import com.falsepattern.zigbrains.zon.psi.ZonTypes;
import com.intellij.psi.tree.TokenSet;

public interface ZonTokenSets {
    TokenSet COMMENTS = TokenSet.create(ZonTypes.COMMENT);
    TokenSet STRINGS = TokenSet.create(ZonTypes.LINE_STRING, ZonTypes.STRING_LITERAL_SINGLE);
}

File: modules/zon/src/main/kotlin/com/falsepattern/zigbrains/zon/parser/ZonTokenSets.kt
package com.falsepattern.zigbrains.zon.parser

import com.falsepattern.zigbrains.zon.psi.ZonTypes
import com.intellij.psi.tree.TokenSet

object ZonTokenSets {
    val COMMENTS = TokenSet.create(ZonTypes.COMMENT)
    val STRINGS = TokenSet.create(ZonTypes.LINE_STRING, ZonTypes.STRING_LITERAL_SINGLE)
}